### PR TITLE
Rare martial art books spawn

### PR DESCRIPTION
**Pull request by ituluwituluwzev**
Books for Capoeira, Shaolin Adept, Venom Mob Protege, and Zui Quan spawn now. They spawn in the same group with The Book of Five Rings and The Modern Swordsman, both of which spawn at roughly 2% in the group. Now, Capoeira book spawns at the same rate, while the overall spawn-rate of other books is only roughly 2%, because these styles are all rare and mysterious.

### CODE REVIEW

File: data/json/item_groups.json [hunk 1]
```diff
@@ -3286,30 +3286,42 @@
...
+      [ "manual_dodge", 15 ],
+      [ "novel_experimental", 24 ],
+      [ "novel_swash", 30 ],
+      [ "manual_fencing", 9 ],
+      [ "manual_niten", 9 ],
+      [ "novel_samurai", 30 ],
+      [ "plays_book", 30 ],
+      [ "poetry_book", 30 ],
+      [ "carpentry_book", 24 ],
+      [ "scots_cookbook", 30 ],
```
Comment (angoddu) [inline data/json/item_groups.json]: Why were these lines (3289-3312) changed? You should only change the lines that you have to for your PR

Comment (ituluwituluwzev) [inline data/json/item_groups.json]: The part before the item list is changed by the web linter. I add 11 new items to the list, so I need larger numbers to keep the frequency. Then the entire list is changed.

Comment (angoddu) [inline data/json/item_groups.json]: Ooh my bad. I didn't notice the numbers changed for the other entries. That makes sense then. I would recommend that the "id" and "type" lines switch place to match the format of the other entries.

Comment (ituluwituluwzev) [inline data/json/item_groups.json]: The web linter made the switch when I input the block. Probably other entries need changing, too.

**Pull request merged**